src/basic/vx_config/config_constants.h

```diff
@@ -1193,6 +1193,18 @@ static const char conf_key_radial_velocity_field_name[] = "radial_velocity_field
 static const char conf_key_tangential_velocity_field_name[] = "tangential_velocity_field_name";
 static const char conf_key_radial_velocity_long_field_name[] = "radial_velocity_long_field_name";
 static const char conf_key_tangential_velocity_long_field_name[] = "tangential_velocity_long_field_name";
+static const char conf_key_vortex_removal[] = "vortex_removal";
+
+//
+// TC-Diag specific paramter key names
```

Comment:
Minor mispelling here: paramter -> parameter

Reply:
Good catch. Its now fixed with this [commit](https://github.com/dtcenter/MET/commit/b6ee12a6c111928772c055105f96b823d1341a20).